### PR TITLE
DHFPROD-4074: Update Versions and Change log

### PR DESCRIPTION
**Pull request by SameeraPriyathamTadikonda**
Please check if the minimum ML version is correct in the READ ME file.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,13 @@
 # Change Log
+## [v5.1.0](https://github.com/marklogic/marklogic-data-hub/releases/tag/v5.2.0) (2020-03-06)
```
Comment (rjrudin) [inline CHANGELOG.md]: Typo, should be 5.2.0

File: README.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ MarkLogic Data Hub is a data integration platform and toolset that helps you qui
 
 # Version Support
 
-For Data Hub version 5.0, you need:
+For Data Hub version 5.2, you need:
```
Comment (rjrudin) [inline README.md]: This information shouldn't be listed here - link to https://docs.marklogic.com/datahub/refs/version-compatibility.html instead.

File: README.md [hunk 3]
```diff
@@ -7,11 +7,11 @@ MarkLogic Data Hub is a data integration platform and toolset that helps you qui
...
 
-  - MarkLogic 9.0-10 or later
   - Gradle 4.6 or later
   - Java JDK 8 or later
+  - Check MarkLogic Version Compatibily [here](https://docs.marklogic.com/datahub/refs/version-compatibility.html)
```
Comment (rjrudin) [inline README.md]: Should be "Compatibility". 

Also, style-wise, I think it's nicer to put "MarkLogic Version Compatibility" in brackets and have the link point to it as opposed to using the word "here".

Comment (SameeraPriyathamTadikonda) [reply README.md]: Okay sure. Will update it.

**Pull request merged**